src/handlers.ts

```diff
@@ -50,7 +50,7 @@ export async function enqueueBuild(req: Request, res: Response, next: NextFuncti
  */
 function assertIdPathParam(req: Request) {
     if (_.isNil(req.params.id)) {
-        throw httpErrors(400, "Must be given an ID")
+        throw httpErrors(400, "Must be given an ID");
```

Comment:
you should be able to use their predefiend bad request error httpErrors.BadRequest("must be given an id")

Reply:
https://www.npmjs.com/package/http-errors